inductiva/molecules/utils.py

```diff
@@ -42,3 +44,19 @@ def align_trajectory_to_average(universe, trajectory_output_path):
                     select="protein and name CA",
                     filename=trajectory_output_path,
                     in_memory=False).run()
+
+
+def download_pdb_from_rcsb(pdb_id: str, save_dir: Optional[str] = None) -> str:
+    """Download a PDB file from the RCSB database according 
+    to its pdb ID.
+    Args:
+        pdb_id: The PDB identifier of the molecule to download.
+    Returns:
+        The path to the downloaded PDB file.
+    """
+    file_path = f"{pdb_id}.pdb"
+    api_url = f"https://files.rcsb.org/download/{file_path}"
```

Comment:
maybe call it rscb_url, because api maybe confused with our API.

Reply:
Indeed, forget to change that. Thanks!